Bugzilla/Auth.pm

```diff
@@ -152,8 +155,11 @@ sub _handle_login_result {
         # because the persistance information can't be re-used again.
         # (See Bugzilla::WebService::Server::JSONRPC for more info.)
         if ($self->{_info_getter}->{successful}->requires_persistence
-            and !Bugzilla->request_cache->{auth_no_automatic_login}) 
-        {
+            and !(
+                Bugzilla->request_cache->{auth_no_automatic_login}
+                || Bugzilla->request_cache->{dont_persist_session}
```

Comment:
I need to remember what dont_persist_session is from. Was that part of the mfa stuff?

Reply:
Maybe it's not used? @quanah

Reply:
Yeah, it's unused @dylanwh .  Updating. :)